File: iniciante/java/1046.java
package iniciante.java;

import java.io.IOException;
import java.util.*;

 
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scanner = new Scanner(System.in);
        int horaInicial = scanner.nextInt();
        int horaFinal = scanner.nextInt();

        System.out.println(String.format("O JOGO DUROU %d HORA(S)",duracaoJogo(horaInicial, horaFinal)));
 
    }
    
    static int duracaoJogo(int a, int b){
        if (a == b)
            return 24;
        if (a > b)
            return 24 - a + b;
        if (a < b)
            return b - a;
        return -1;
    }
 
}

File: iniciante/python/1046.py
# -*- coding: utf-8 -*-

entrada = input().split(' ')
entrada = [int(x) for x in entrada]

def calculo_hora(a, b):
    count_hora = 0
    while True:
        a +=1
        count_hora +=1
        if a == 24:
            a = 0
        if a == b:
            break
    return count_hora

if entrada[0] == entrada[1]:
    print('O JOGO DUROU 24 HORA(S)')
else:
    print(f'O JOGO DUROU {calculo_hora(entrada[0], entrada[1])} HORA(S)')
